
RPM's brp checks say:

ERROR: RPATH "/home/abuild/rpmbuild/BUILD/libseccomp-2.2.0/src/.libs"
on <buildroot>/usr/bin/scmp_sys_resolver is not allowed

---
 tools/Makefile.am |    2 --
 1 file changed, 2 deletions(-)

Index: libseccomp-2.4.0/tools/Makefile.am
===================================================================
--- libseccomp-2.4.0.orig/tools/Makefile.am
+++ libseccomp-2.4.0/tools/Makefile.am
@@ -37,9 +37,7 @@ scmp_bpf_sim_SOURCES = scmp_bpf_sim.c bp
 scmp_api_level_SOURCES = scmp_api_level.c
 
 scmp_sys_resolver_LDADD = ../src/libseccomp.la
-scmp_sys_resolver_LDFLAGS = -static
 scmp_arch_detect_LDADD = ../src/libseccomp.la
-scmp_arch_detect_LDFLAGS = -static
 scmp_bpf_disasm_LDADD = util.la
 scmp_bpf_sim_LDADD = util.la
 scmp_api_level_LDADD = ../src/libseccomp.la
